cola/themes.py

```diff
@@ -109,371 +108,371 @@ def style_sheet_flat(self):
+                border-color: {darker} {field} {field} {field};
                 border-width: 4px 3px 0 3px;
-            }
-            QAbstractSpinBox::down-arrow:hover {
-                border-color: %(button)s %(field)s %(field)s %(field)s;
+            }}
+            QAbstractSpinBox::down-arrow:hover {{
+                border-color: {button} {field} {field} {field};
                 border-width: 4px 3px 0 3px;
-            }
+            }}
 
             /* dialogs */
-            QDialog > QFrame {
+            QDialog > QFrame {{
                 margin: 2px 2px 2px 2px;
-            }
+            }}
 
             /* headers */
-            QHeaderView {
-                color: %(field_text)s;
+            QHeaderView {{
+                color: {field_text};
                 border-style: solid;
                 border-width: 0 0 1px 0;
-                border-color: %(grayed)s;
-            }
-            QHeaderView::section {
+                border-color: {grayed};
+            }}
+            QHeaderView::section {{
                 border-style: solid;
-                border-right: 1px solid %(grayed)s;
-                background-color: %(background)s;
-                color: %(field_text)s;
+                border-right: 1px solid {grayed};
+                background-color: {background};
+                color: {field_text};
                 padding-left: 4px;
-            }
+            }}
 
             /* headers */
-            QHeaderView {
-                color: %(field_text)s;
+            QHeaderView {{
+                color: {field_text};
                 border-style: solid;
                 border-width: 0 0 1px 0;
-                border-color: %(grayed)s;
-            }
-            QHeaderView::section {
+                border-color: {grayed};
+            }}
+            QHeaderView::section {{
                 border-style: solid;
-                border-right: 1px solid %(grayed)s;
-                background-color: %(background)s;
-                color: %(field_text)s;
+                border-right: 1px solid {grayed};
+                background-color: {background};
+                color: {field_text};
                 padding-left: 4px;
-            }
-
-            """ % {
-            'background': background,
-            'field': field,
-            'button': color_rgb,
-            'darker': darker,
-            'lighter': lighter,
-            'grayed': grayed,
-            'button_text': button_text,
-            'field_text': field_text,
-            'separator': defs.separator,
-            'focus': focus,
-        }
+            }}
+
+            """.format(
+            background=background,
+            field=field,
+            button=color_rgb,
+            darker=darker,
+            lighter=lighter,
+            grayed=grayed,
+            button_text=button_text,
+            field_text=field_text,
+            separator=defs.separator,
+            focus=focus,
+        )
```

Comment:
this is ok, but the stylesheets are arguably nicer without the f-string hammer due to the wide use of `{ }` in qss.